app/controllers/tastings_controller.rb

```diff
@@ -26,6 +26,14 @@ def create
     @tasting.host = current_user
     authorize @tasting
     if @tasting.save!
+      assign_categorization(@tasting)
+      Participation.create!(
```

Comment:
We should use save and create instead of save! and create! to make sure it returns false and goes to the else option if it fails.
Couple of lines to update

Reply:
Good call! Just made requested changes